### PR TITLE
Apply timeline copy updates

### PR DESCRIPTION
**Pull request by kategarklavs**
- From step 1, removed "make sure your company is incorporated" from heading (per Ben's suggestion).
- Updated Eligibility Guide link to SBIR site link (replaced PDF link that I'd previously added)
- Adding TTA section intro

### CODE REVIEW

**Comment by kategarklavs** [comment]
@femmebot & @line47 - Just made a few small edits and added an intro to the TTA section. Let me know what you think!

**Comment by kategarklavs** [comment]
@femmebot  - I tagged you for review in this one because the content change is slightly more than minor (added the TTA section intro) 👍

File: _timeline/step-1.md [hunk 1]
```diff
@@ -17,14 +17,16 @@ Here are the criteria we use to determine which companies to fund:
 - **Seed stage:** Your small business is working on a new product or service enabled by innovative, early-stage technology.
 - **Pre-traction:** You have evidence that your product or service could meet an important, unmet need for your customers.
 - **High-risk:** Your product is based on unproven technology that needs further testing (and funding for that testing).
-- **Scale:** If you successfully bring your product to market, it could form the foundation for a scalable business and make a large impact in your target market.
+- **Scale:** If you successfully bring your product or service to market, it could form the foundation for a scalable business and make a large impact in your target market.
 
 **Portfolio**
 
-You can check out our [portfolio](#) to see what companies we’ve recently funded.
+Check out our [portfolio](#) to see what companies we’ve recently funded.
```
Comment (line47) [inline _timeline/step-1.md]: We should like this portfolio page

Comment (gemfarmer) [inline _timeline/step-1.md]:
```
[portfolio]({{ site.baseurl }}/portfolio/)
```

**Pull request merged**